src/Storage/Controllers/InstancesController.cs

```diff
@@ -118,13 +118,59 @@ public async Task<ActionResult<QueryResponse<Instance>>> GetInstances(InstanceQu
                 {
                     queryParameters.Org = queryParameters.AppId.Split('/')[0];
                 }
-
+                
                 if (!orgClaim.Equals(queryParameters.Org, StringComparison.InvariantCultureIgnoreCase))
                 {
-                    return Forbid();
-                }
+                    var appId = ValidateAppId(queryParameters.AppId).Split("/")[1];
+                    XacmlJsonRequestRoot request;
+                    try
+                    {
+                        var instanceOwnerPartyId = (int)queryParameters.InstanceOwnerPartyId;
```

Comment:
Instance owner will usually be undefined when the user is an application owner (org).

Reply:
I think the cast will fail if party id is missing and that is the most common case. Further down in the controller we also have more party id logic where we try to convert organization number or national identity number to a party id. Not very common, but too late for the check being done here. 

Application owner will never be given any type or role for an instance owner. I believe its safe to exclude instance owner party id from the authorization request where we know that "access-subject" is an application owner.

Reply:
`DecisionHelper.CreateDecisionRequest` expects a `instanceOwnerPartyId`. Do you have any suggestions as to how to solve this? Will i have to go into Altinn.Common and add a method for creating a `DecisionRequest` without passing `instanceOwnerPartyId`?

Reply:
Looks like we would need to expand AuthorizationService with a new method for this case.

Reply:
Fixed with new Altinn.Common.PEP v4.1.2